src/components/Modal/BasicModal/BasicModal.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { Button } from "components/Button";
+import { BasicModal, BasicModalProps } from "./BasicModal";
+import useModal from "../useModal";
+
+export default {
+  title: "Components/Basic Modal",
```

Comment:
All modals should be under `Modal`, just like the folder structure 
```suggestion
  title: "Components/Modal/Basic",
```